org.flathub.electron-sample-app.yml

```diff
@@ -0,0 +1,56 @@
+app-id: org.flathub.electron-sample-app
```

Comment:
why this file?

Reply:
Accidentally added when moving b/w my personal git repo I used while testing.  Will remove.